### DISCUSSION TITLE
Is it possible to conditionally render something only on the xth page?

### DISCUSSION THREAD

**Discussion by eblinge** [wojtekmaj/react-pdf#1442] [Q&A]
Hi there!

I currently have a PDF with dynamic content, part of which is a table. The table is a View that consists of Text elements that is stylized to look like a table. There may be certain situations where I have a table that spans across multiple pages, and if this happens I'm trying to add text to the table title that says "Continued." I haven't been able to find a way to render something, only if it spans across multiple pages. Does anyone know if this is something that is possible? 

Thanks so much!!

**Comment by wojtekmaj** [comment]
Shouldn't be a problem, here's a modified example I used for watermarks, but applied only to one page:

https://codesandbox.io/s/react-pdf-watermark-first-page-only-rjxj2r?file=/src/index.jsx

**Reply by eblinge** [reply]
Thank you for the response! I'm not sure if this is a route I can take, though. I should have specified in my original question that I do not know which pages the tables are going to be on. Some of the documents have 3 tables, for example, so the first one might span from pages 1-3, and the second table may start anywhere after that point. So I never know which specific pages the elements are on.

**Reply by wojtekmaj** [reply]
I'm re-reading your question and I'm more convinced now that you actually may be looking for support with @react-pdf/renderer.
